[2024-10-11T23:34:10.268+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-11T23:34:10.298+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: predictions_lstm_dag_v1.predictions_task_venv manual__2024-10-11T23:34:08.652282+00:00 [queued]>
[2024-10-11T23:34:10.308+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: predictions_lstm_dag_v1.predictions_task_venv manual__2024-10-11T23:34:08.652282+00:00 [queued]>
[2024-10-11T23:34:10.310+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-10-11T23:34:10.340+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): predictions_task_venv> on 2024-10-11 23:34:08.652282+00:00
[2024-10-11T23:34:10.354+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=51) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-11T23:34:10.358+0000] {standard_task_runner.py:63} INFO - Started process 53 to run task
[2024-10-11T23:34:10.364+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'predictions_lstm_dag_v1', 'predictions_task_venv', 'manual__2024-10-11T23:34:08.652282+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/prediction_dag.py', '--cfg-path', '/tmp/tmpq2qkhr28']
[2024-10-11T23:34:10.367+0000] {standard_task_runner.py:91} INFO - Job 55: Subtask predictions_task_venv
[2024-10-11T23:34:10.476+0000] {task_command.py:426} INFO - Running <TaskInstance: predictions_lstm_dag_v1.predictions_task_venv manual__2024-10-11T23:34:08.652282+00:00 [running]> on host 8a428b66f6b4
[2024-10-11T23:34:10.621+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Michael Mora' AIRFLOW_CTX_DAG_ID='predictions_lstm_dag_v1' AIRFLOW_CTX_TASK_ID='predictions_task_venv' AIRFLOW_CTX_EXECUTION_DATE='2024-10-11T23:34:08.652282+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-11T23:34:08.652282+00:00'
[2024-10-11T23:34:10.642+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-11T23:34:10.646+0000] {process_utils.py:183} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/venvbkf5he8t --system-site-packages --python=python
[2024-10-11T23:34:10.648+0000] {process_utils.py:187} INFO - Output:
[2024-10-11T23:34:11.328+0000] {process_utils.py:191} INFO - created virtual environment CPython3.12.3.final.0-64 in 257ms
[2024-10-11T23:34:11.334+0000] {process_utils.py:191} INFO -   creator CPython3Posix(dest=/tmp/venvbkf5he8t, clear=False, no_vcs_ignore=False, global=True)
[2024-10-11T23:34:11.336+0000] {process_utils.py:191} INFO -   seeder FromAppData(download=False, pip=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2024-10-11T23:34:11.337+0000] {process_utils.py:191} INFO -     added seed packages: pip==24.2
[2024-10-11T23:34:11.337+0000] {process_utils.py:191} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-10-11T23:34:11.352+0000] {process_utils.py:183} INFO - Executing cmd: /tmp/venvbkf5he8t/bin/pip install -r /tmp/venvbkf5he8t/requirements.txt
[2024-10-11T23:34:11.353+0000] {process_utils.py:187} INFO - Output:
[2024-10-11T23:34:12.460+0000] {process_utils.py:191} INFO - Collecting joblib (from -r /tmp/venvbkf5he8t/requirements.txt (line 1))
[2024-10-11T23:34:12.462+0000] {process_utils.py:191} INFO -   Using cached joblib-1.4.2-py3-none-any.whl.metadata (5.4 kB)
[2024-10-11T23:34:12.485+0000] {process_utils.py:191} INFO - Collecting keras (from -r /tmp/venvbkf5he8t/requirements.txt (line 2))
[2024-10-11T23:34:12.486+0000] {process_utils.py:191} INFO -   Using cached keras-3.6.0-py3-none-any.whl.metadata (5.8 kB)
[2024-10-11T23:34:12.487+0000] {process_utils.py:191} INFO - Requirement already satisfied: numpy in /home/***/.local/lib/python3.12/site-packages (from -r /tmp/venvbkf5he8t/requirements.txt (line 3)) (1.26.4)
[2024-10-11T23:34:12.489+0000] {process_utils.py:191} INFO - Requirement already satisfied: pandas in /home/***/.local/lib/python3.12/site-packages (from -r /tmp/venvbkf5he8t/requirements.txt (line 4)) (2.1.4)
[2024-10-11T23:34:12.553+0000] {process_utils.py:191} INFO - Collecting scikit-learn (from -r /tmp/venvbkf5he8t/requirements.txt (line 5))
[2024-10-11T23:34:12.556+0000] {process_utils.py:191} INFO -   Using cached scikit_learn-1.5.2-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (13 kB)
[2024-10-11T23:34:12.657+0000] {process_utils.py:191} INFO - Collecting scipy (from -r /tmp/venvbkf5he8t/requirements.txt (line 6))
[2024-10-11T23:34:12.659+0000] {process_utils.py:191} INFO -   Using cached scipy-1.14.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (113 kB)
[2024-10-11T23:34:12.772+0000] {process_utils.py:191} INFO - Collecting tensorflow (from -r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:12.774+0000] {process_utils.py:191} INFO -   Using cached tensorflow-2.17.0-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (4.1 kB)
[2024-10-11T23:34:12.809+0000] {process_utils.py:191} INFO - Collecting absl-py (from keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2))
[2024-10-11T23:34:12.811+0000] {process_utils.py:191} INFO -   Using cached absl_py-2.1.0-py3-none-any.whl.metadata (2.3 kB)
[2024-10-11T23:34:12.812+0000] {process_utils.py:191} INFO - Requirement already satisfied: rich in /home/***/.local/lib/python3.12/site-packages (from keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2)) (13.7.1)
[2024-10-11T23:34:12.824+0000] {process_utils.py:191} INFO - Collecting namex (from keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2))
[2024-10-11T23:34:12.825+0000] {process_utils.py:191} INFO -   Using cached namex-0.0.8-py3-none-any.whl.metadata (246 bytes)
[2024-10-11T23:34:12.878+0000] {process_utils.py:191} INFO - Collecting h5py (from keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2))
[2024-10-11T23:34:12.879+0000] {process_utils.py:191} INFO -   Using cached h5py-3.12.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (2.5 kB)
[2024-10-11T23:34:12.920+0000] {process_utils.py:191} INFO - Collecting optree (from keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2))
[2024-10-11T23:34:12.921+0000] {process_utils.py:191} INFO -   Using cached optree-0.13.0-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (47 kB)
[2024-10-11T23:34:13.002+0000] {process_utils.py:191} INFO - Collecting ml-dtypes (from keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2))
[2024-10-11T23:34:13.004+0000] {process_utils.py:191} INFO -   Using cached ml_dtypes-0.5.0-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (21 kB)
[2024-10-11T23:34:13.008+0000] {process_utils.py:191} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.12/site-packages (from keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2)) (24.0)
[2024-10-11T23:34:13.015+0000] {process_utils.py:191} INFO - Requirement already satisfied: python-dateutil>=2.8.2 in /home/***/.local/lib/python3.12/site-packages (from pandas->-r /tmp/venvbkf5he8t/requirements.txt (line 4)) (2.9.0.post0)
[2024-10-11T23:34:13.016+0000] {process_utils.py:191} INFO - Requirement already satisfied: pytz>=2020.1 in /home/***/.local/lib/python3.12/site-packages (from pandas->-r /tmp/venvbkf5he8t/requirements.txt (line 4)) (2024.1)
[2024-10-11T23:34:13.016+0000] {process_utils.py:191} INFO - Requirement already satisfied: tzdata>=2022.1 in /home/***/.local/lib/python3.12/site-packages (from pandas->-r /tmp/venvbkf5he8t/requirements.txt (line 4)) (2024.1)
[2024-10-11T23:34:13.035+0000] {process_utils.py:191} INFO - Collecting threadpoolctl>=3.1.0 (from scikit-learn->-r /tmp/venvbkf5he8t/requirements.txt (line 5))
[2024-10-11T23:34:13.036+0000] {process_utils.py:191} INFO -   Using cached threadpoolctl-3.5.0-py3-none-any.whl.metadata (13 kB)
[2024-10-11T23:34:13.106+0000] {process_utils.py:191} INFO - Collecting astunparse>=1.6.0 (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:13.108+0000] {process_utils.py:191} INFO -   Using cached astunparse-1.6.3-py2.py3-none-any.whl.metadata (4.4 kB)
[2024-10-11T23:34:13.142+0000] {process_utils.py:191} INFO - Collecting flatbuffers>=24.3.25 (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:13.144+0000] {process_utils.py:191} INFO -   Using cached flatbuffers-24.3.25-py2.py3-none-any.whl.metadata (850 bytes)
[2024-10-11T23:34:13.170+0000] {process_utils.py:191} INFO - Collecting gast!=0.5.0,!=0.5.1,!=0.5.2,>=0.2.1 (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:13.171+0000] {process_utils.py:191} INFO -   Using cached gast-0.6.0-py3-none-any.whl.metadata (1.3 kB)
[2024-10-11T23:34:13.213+0000] {process_utils.py:191} INFO - Collecting google-pasta>=0.1.1 (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:13.214+0000] {process_utils.py:191} INFO -   Using cached google_pasta-0.2.0-py3-none-any.whl.metadata (814 bytes)
[2024-10-11T23:34:13.237+0000] {process_utils.py:191} INFO - Collecting libclang>=13.0.0 (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:13.238+0000] {process_utils.py:191} INFO -   Using cached libclang-18.1.1-py2.py3-none-manylinux2014_aarch64.whl.metadata (5.2 kB)
[2024-10-11T23:34:13.240+0000] {process_utils.py:191} INFO - Collecting ml-dtypes (from keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2))
[2024-10-11T23:34:13.241+0000] {process_utils.py:191} INFO -   Using cached ml_dtypes-0.4.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (20 kB)
[2024-10-11T23:34:13.259+0000] {process_utils.py:191} INFO - Collecting opt-einsum>=2.3.2 (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:13.260+0000] {process_utils.py:191} INFO -   Using cached opt_einsum-3.4.0-py3-none-any.whl.metadata (6.3 kB)
[2024-10-11T23:34:13.261+0000] {process_utils.py:191} INFO - Requirement already satisfied: protobuf!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.20.3 in /home/***/.local/lib/python3.12/site-packages (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (4.25.3)
[2024-10-11T23:34:13.262+0000] {process_utils.py:191} INFO - Requirement already satisfied: requests<3,>=2.21.0 in /home/***/.local/lib/python3.12/site-packages (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (2.31.0)
[2024-10-11T23:34:13.263+0000] {process_utils.py:191} INFO - Requirement already satisfied: setuptools in /home/***/.local/lib/python3.12/site-packages (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (66.1.1)
[2024-10-11T23:34:13.263+0000] {process_utils.py:191} INFO - Requirement already satisfied: six>=1.12.0 in /home/***/.local/lib/python3.12/site-packages (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (1.16.0)
[2024-10-11T23:34:13.264+0000] {process_utils.py:191} INFO - Requirement already satisfied: termcolor>=1.1.0 in /home/***/.local/lib/python3.12/site-packages (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (2.4.0)
[2024-10-11T23:34:13.264+0000] {process_utils.py:191} INFO - Requirement already satisfied: typing-extensions>=3.6.6 in /home/***/.local/lib/python3.12/site-packages (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (4.11.0)
[2024-10-11T23:34:13.265+0000] {process_utils.py:191} INFO - Requirement already satisfied: wrapt>=1.11.0 in /home/***/.local/lib/python3.12/site-packages (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (1.16.0)
[2024-10-11T23:34:13.265+0000] {process_utils.py:191} INFO - Requirement already satisfied: grpcio<2.0,>=1.24.3 in /home/***/.local/lib/python3.12/site-packages (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (1.63.0)
[2024-10-11T23:34:13.305+0000] {process_utils.py:191} INFO - Collecting tensorboard<2.18,>=2.17 (from tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:13.306+0000] {process_utils.py:191} INFO -   Using cached tensorboard-2.17.1-py3-none-any.whl.metadata (1.6 kB)
[2024-10-11T23:34:13.311+0000] {process_utils.py:191} INFO - Requirement already satisfied: wheel<1.0,>=0.23.0 in /usr/local/lib/python3.12/site-packages (from astunparse>=1.6.0->tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (0.43.0)
[2024-10-11T23:34:13.318+0000] {process_utils.py:191} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.12/site-packages (from requests<3,>=2.21.0->tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (3.3.2)
[2024-10-11T23:34:13.319+0000] {process_utils.py:191} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.12/site-packages (from requests<3,>=2.21.0->tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (3.7)
[2024-10-11T23:34:13.320+0000] {process_utils.py:191} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.12/site-packages (from requests<3,>=2.21.0->tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (2.2.1)
[2024-10-11T23:34:13.320+0000] {process_utils.py:191} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.12/site-packages (from requests<3,>=2.21.0->tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (2024.2.2)
[2024-10-11T23:34:13.362+0000] {process_utils.py:191} INFO - Collecting markdown>=2.6.8 (from tensorboard<2.18,>=2.17->tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:13.363+0000] {process_utils.py:191} INFO -   Using cached Markdown-3.7-py3-none-any.whl.metadata (7.0 kB)
[2024-10-11T23:34:13.394+0000] {process_utils.py:191} INFO - Collecting tensorboard-data-server<0.8.0,>=0.7.0 (from tensorboard<2.18,>=2.17->tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7))
[2024-10-11T23:34:13.396+0000] {process_utils.py:191} INFO -   Using cached tensorboard_data_server-0.7.2-py3-none-any.whl.metadata (1.1 kB)
[2024-10-11T23:34:13.397+0000] {process_utils.py:191} INFO - Requirement already satisfied: werkzeug>=1.0.1 in /home/***/.local/lib/python3.12/site-packages (from tensorboard<2.18,>=2.17->tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (2.2.3)
[2024-10-11T23:34:13.406+0000] {process_utils.py:191} INFO - Requirement already satisfied: markdown-it-py>=2.2.0 in /home/***/.local/lib/python3.12/site-packages (from rich->keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2)) (3.0.0)
[2024-10-11T23:34:13.407+0000] {process_utils.py:191} INFO - Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /home/***/.local/lib/python3.12/site-packages (from rich->keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2)) (2.17.2)
[2024-10-11T23:34:13.412+0000] {process_utils.py:191} INFO - Requirement already satisfied: mdurl~=0.1 in /home/***/.local/lib/python3.12/site-packages (from markdown-it-py>=2.2.0->rich->keras->-r /tmp/venvbkf5he8t/requirements.txt (line 2)) (0.1.2)
[2024-10-11T23:34:13.415+0000] {process_utils.py:191} INFO - Requirement already satisfied: MarkupSafe>=2.1.1 in /home/***/.local/lib/python3.12/site-packages (from werkzeug>=1.0.1->tensorboard<2.18,>=2.17->tensorflow->-r /tmp/venvbkf5he8t/requirements.txt (line 7)) (2.1.5)
[2024-10-11T23:34:13.419+0000] {process_utils.py:191} INFO - Using cached joblib-1.4.2-py3-none-any.whl (301 kB)
[2024-10-11T23:34:13.421+0000] {process_utils.py:191} INFO - Using cached keras-3.6.0-py3-none-any.whl (1.2 MB)
[2024-10-11T23:34:13.425+0000] {process_utils.py:191} INFO - Using cached scikit_learn-1.5.2-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (12.1 MB)
[2024-10-11T23:34:13.457+0000] {process_utils.py:191} INFO - Using cached scipy-1.14.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (35.3 MB)
[2024-10-11T23:34:13.546+0000] {process_utils.py:191} INFO - Using cached tensorflow-2.17.0-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (224.0 MB)
[2024-10-11T23:34:14.294+0000] {process_utils.py:191} INFO - Using cached absl_py-2.1.0-py3-none-any.whl (133 kB)
[2024-10-11T23:34:14.301+0000] {process_utils.py:191} INFO - Using cached astunparse-1.6.3-py2.py3-none-any.whl (12 kB)
[2024-10-11T23:34:14.303+0000] {process_utils.py:191} INFO - Using cached flatbuffers-24.3.25-py2.py3-none-any.whl (26 kB)
[2024-10-11T23:34:14.306+0000] {process_utils.py:191} INFO - Using cached gast-0.6.0-py3-none-any.whl (21 kB)
[2024-10-11T23:34:14.308+0000] {process_utils.py:191} INFO - Using cached google_pasta-0.2.0-py3-none-any.whl (57 kB)
[2024-10-11T23:34:14.310+0000] {process_utils.py:191} INFO - Using cached h5py-3.12.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (5.2 MB)
[2024-10-11T23:34:14.322+0000] {process_utils.py:191} INFO - Using cached libclang-18.1.1-py2.py3-none-manylinux2014_aarch64.whl (23.8 MB)
[2024-10-11T23:34:14.379+0000] {process_utils.py:191} INFO - Using cached ml_dtypes-0.4.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (2.2 MB)
[2024-10-11T23:34:14.389+0000] {process_utils.py:191} INFO - Using cached opt_einsum-3.4.0-py3-none-any.whl (71 kB)
[2024-10-11T23:34:14.393+0000] {process_utils.py:191} INFO - Using cached tensorboard-2.17.1-py3-none-any.whl (5.5 MB)
[2024-10-11T23:34:14.413+0000] {process_utils.py:191} INFO - Using cached threadpoolctl-3.5.0-py3-none-any.whl (18 kB)
[2024-10-11T23:34:14.419+0000] {process_utils.py:191} INFO - Using cached namex-0.0.8-py3-none-any.whl (5.8 kB)
[2024-10-11T23:34:14.445+0000] {process_utils.py:191} INFO - Using cached optree-0.13.0-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (332 kB)
[2024-10-11T23:34:14.449+0000] {process_utils.py:191} INFO - Using cached Markdown-3.7-py3-none-any.whl (106 kB)
[2024-10-11T23:34:14.452+0000] {process_utils.py:191} INFO - Using cached tensorboard_data_server-0.7.2-py3-none-any.whl (2.4 kB)
[2024-10-11T23:34:15.714+0000] {process_utils.py:191} INFO - Installing collected packages: namex, libclang, flatbuffers, threadpoolctl, tensorboard-data-server, scipy, optree, opt-einsum, ml-dtypes, markdown, joblib, h5py, google-pasta, gast, astunparse, absl-py, tensorboard, scikit-learn, keras, tensorflow
[2024-10-11T23:34:31.289+0000] {process_utils.py:191} INFO - Successfully installed absl-py-2.1.0 astunparse-1.6.3 flatbuffers-24.3.25 gast-0.6.0 google-pasta-0.2.0 h5py-3.12.1 joblib-1.4.2 keras-3.6.0 libclang-18.1.1 markdown-3.7 ml-dtypes-0.4.1 namex-0.0.8 opt-einsum-3.4.0 optree-0.13.0 scikit-learn-1.5.2 scipy-1.14.1 tensorboard-2.17.1 tensorboard-data-server-0.7.2 tensorflow-2.17.0 threadpoolctl-3.5.0
[2024-10-11T23:34:31.992+0000] {process_utils.py:183} INFO - Executing cmd: /tmp/venvbkf5he8t/bin/python /tmp/venv-callar1ru7hc/script.py /tmp/venv-callar1ru7hc/script.in /tmp/venv-callar1ru7hc/script.out /tmp/venv-callar1ru7hc/string_args.txt /tmp/venv-callar1ru7hc/termination.log
[2024-10-11T23:34:31.999+0000] {process_utils.py:187} INFO - Output:
[2024-10-11T23:34:36.598+0000] {process_utils.py:191} INFO - Traceback (most recent call last):
[2024-10-11T23:34:36.601+0000] {process_utils.py:191} INFO -   File "/tmp/venv-callar1ru7hc/script.py", line 56, in <module>
[2024-10-11T23:34:36.602+0000] {process_utils.py:191} INFO -     res = run_predictions_task(*arg_dict["args"], **arg_dict["kwargs"])
[2024-10-11T23:34:36.602+0000] {process_utils.py:191} INFO -           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-10-11T23:34:36.603+0000] {process_utils.py:191} INFO -   File "/tmp/venv-callar1ru7hc/script.py", line 21, in run_predictions_task
[2024-10-11T23:34:36.604+0000] {process_utils.py:191} INFO -     model_file = load_model("/opt/***/dags/models/model_net_lstm.keras'")
[2024-10-11T23:34:36.605+0000] {process_utils.py:191} INFO -                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-10-11T23:34:36.606+0000] {process_utils.py:191} INFO -   File "/tmp/venvbkf5he8t/lib/python3.12/site-packages/keras/src/saving/saving_api.py", line 206, in load_model
[2024-10-11T23:34:36.607+0000] {process_utils.py:191} INFO -     raise ValueError(
[2024-10-11T23:34:36.608+0000] {process_utils.py:191} INFO - ValueError: File format not supported: filepath=/opt/***/dags/models/model_net_lstm.keras'. Keras 3 only supports V3 `.keras` files and legacy H5 format files (`.h5` extension). Note that the legacy SavedModel format is not supported by `load_model()` in Keras 3. In order to reload a TensorFlow SavedModel as an inference-only layer in Keras 3, use `keras.layers.TFSMLayer(/opt/***/dags/models/model_net_lstm.keras', call_endpoint='serving_default')` (note that your `call_endpoint` might have a different name).
[2024-10-11T23:34:38.679+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-11T23:34:38.704+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/decorators/base.py", line 265, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 437, in execute
    return super().execute(context=serializable_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 759, in execute_callable
    result = self._execute_python_callable_in_subprocess(python_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 514, in _execute_python_callable_in_subprocess
    raise AirflowException(error_msg) from None
airflow.exceptions.AirflowException: Process returned non-zero exit status 1.
File format not supported: filepath=/opt/***/dags/models/model_net_lstm.keras'. Keras 3 only supports V3 `.keras` files and legacy H5 format files (`.h5` extension). Note that the legacy SavedModel format is not supported by `load_model()` in Keras 3. In order to reload a TensorFlow SavedModel as an inference-only layer in Keras 3, use `keras.layers.TFSMLayer(/opt/***/dags/models/model_net_lstm.keras', call_endpoint='serving_default')` (note that your `call_endpoint` might have a different name).
[2024-10-11T23:34:38.718+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=predictions_lstm_dag_v1, task_id=predictions_task_venv, run_id=manual__2024-10-11T23:34:08.652282+00:00, execution_date=20241011T233408, start_date=20241011T233410, end_date=20241011T233438
[2024-10-11T23:34:38.737+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 55 for task predictions_task_venv (Process returned non-zero exit status 1.
File format not supported: filepath=/opt/***/dags/models/model_net_lstm.keras'. Keras 3 only supports V3 `.keras` files and legacy H5 format files (`.h5` extension). Note that the legacy SavedModel format is not supported by `load_model()` in Keras 3. In order to reload a TensorFlow SavedModel as an inference-only layer in Keras 3, use `keras.layers.TFSMLayer(/opt/***/dags/models/model_net_lstm.keras', call_endpoint='serving_default')` (note that your `call_endpoint` might have a different name).; 53)
[2024-10-11T23:34:38.775+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-11T23:34:38.797+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-11T23:34:38.798+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
